python3.8/cookiecutter-aws-sam-eventBridge-python/{{cookiecutter.project_name}}/tests/integration/test_ec2_event.py

```diff
@@ -0,0 +1,100 @@
+import logging
+import os
+from time import sleep, time
+from unittest import TestCase
+
+import boto3
+from botocore.exceptions import ClientError
+
+"""
+Make sure env variable AWS_SAM_STACK_NAME exists with the name of the stack we are going to test. 
+"""
+
+
+class TestEC2Event(TestCase):
+    """
+    This integration test will create an EC2 and verify the lambda function is invoked by checking the cloudwatch log
+    The EC2 instance will be deleted when test completes.
+    """
+
+    function_name: str
+    instance_id: str  # temporary EC2 instance ID
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        stack_name = os.environ.get("AWS_SAM_STACK_NAME")
+        if not stack_name:
```

Comment:
There is some minor inconsistency in this class with previous one? Can we not abstract out fetching stack name in a separate method, which validates if name exist and fails with more descriptive message?